code/CODE.ino

```diff
@@ -0,0 +1,113 @@
+#include <PS4Controller.h>
```

Comment:
First line comment should say something like:

`// code source: https://github.com/un0038998/PS4Controller_Tank`